util/db/secrets.go

```diff
@@ -155,8 +159,20 @@ func URIToSecretName(uriType, uri string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	re := regexp.MustCompile(`:\d+$`)
+	host := re.ReplaceAllLiteralString(parsedURI.Host, "")
+	if strings.Contains(host, ":") {
+		addr, err := netip.ParseAddr(host[1 : len(host)-1])
+		if err != nil {
+			return "", err
+		}
+		host = strings.ReplaceAll(addr.StringExpanded(), ":", "-")
+	}
+	length := len(host) + len(uriType) + 12
+	if length > 63 {
+		host = host[0 : len(host)-(length-63)]
```

Comment:
This will crash if `uriType` is really long and `host` is really short. In general, I do not understand how `length` is calculated here.

Reply:
Thanks for your comment. I removed these codes, the secret name length is limited to 253, it's long enough.